docs/content/users/install/ddev-installation.md

```diff
@@ -114,9 +114,38 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
     * Installing Chocolatey package manager (optional).
     * One time initialization of mkcert.
     * Installing WSL2 and installing a distro like Ubuntu.
-    * Installing or upgrading to the latest Docker Desktop for Windows with WSL2 enabled.
+    * (Optionally) Installing Docker Desktop for Windows and enabling WSL2 integration with the distro.
     * Installing DDEV inside your distro.
 
+    ### WSL2 with Docker-ce Inside Scripted Installation
+
+    This scripted installation prepares your default WSL2 Ubuntu distro for use and has no dependency on Docker Desktop.
+
+    You can do these things manually, or you can do most of it with the provided PowerShell script. 
+    In all cases:
+    
+    1. Install WSL2 with an Ubuntu distro. On a system without WSL2, just run:
+        ```powershell
+        wsl --install
+        ```
+
+        Verify that you have an Ubuntu distro set to default with `wsl -l -v`.
+
+        If you already have WSL2 but don't have an Ubuntu distro, install one with `wsl --install Ubuntu`. 
+
+        If that doesn't work for you, see the [manual installation](https://docs.microsoft.com/en-us/windows/wsl/install-manual) and linked [troubleshooting](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting#installation-issues).
+        
+        If you prefer to use another Ubuntu distro, just install it and set it as default. For example, `wsl --set-default Ubuntu-22.04`.
```

Comment:
[Omit “just”](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#avoid-just-and-easy):

```suggestion
        If you prefer to use another Ubuntu distro, install it and set it as default. For example, `wsl --set-default Ubuntu-22.04`.
```